packages/editor/src/store/test/selectors.js

```diff
@@ -174,6 +174,7 @@ describe( 'selectors', () => {
 		setFreeformContentHandlerName( 'core/test-freeform' );
 
 		cachedSelectors.forEach( ( { clear } ) => clear() );
+		invoke( getEditedPostAttribute.mergeCache, [ 'clear' ] );
```

Comment:
I guess this doesn't exist:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap/clear

Given how the tests are written, it might not really be necessary to clear the cache. Alternatively, we could manually assign into `getEditedPostAttribute` a new `WeakMap` in `beforeEach`.